lib/beaker/host_prebuilt_steps.rb

```diff
@@ -502,7 +502,7 @@ def set_env host, opts
           host.exec(Command.new("service ssh restart"))
         when /el-|centos|fedora|redhat|oracle|scientific|eos/
           host.exec(Command.new("echo '\nPermitUserEnvironment yes' >> /etc/ssh/sshd_config"))
-          host.exec(Command.new("/sbin/service sshd restart"))
+          host.exec(Command.new("/sbin/service sshd restart")) unless host['hypervisor'] == 'docker'
```

Comment:
If you are making hypervisor specific alterations to the workflow then it should occur in the docker.rb file by overloading the configuration/validation steps that occur post-provisioning.